### ISSUE TITLE
Encoding problem with °

### ISSUE THREAD

**Issue by tbeu** [RWTH-EBC/AixLib#155]
The file
```
z:\AixLib\AixLib\HVAC\AirHandlingUnit\AHU.mo
z:\AixLib\AixLib\Building\Components\Weather\RadiationOnTiltedSurface\RadOnTiltedSurf_Perez.mo
z:\AixLib\AixLib\Building\Components\Weather\BaseClasses\Sun.mo
```
contain the `°` symbol in ANSI encoding and should be converted to either ASCII or UTF8.

**Comment by PRemmen** [comment]
closed by #159

**Comment by tbeu** [comment]
I observe that the newly changed files
```
f:\Libraries\AixLib\AixLib\HVAC\AirHandlingUnit\AHU.mo
f:\Libraries\AixLib\AixLib\Building\Components\Weather\RadiationOnTiltedSurface\RadOnTiltedSurf_Perez.mo
f:\Libraries\AixLib\AixLib\Building\Components\Weather\BaseClasses\Sun.mo
```
are now UTF8 encoded **with** BOM whereas
```
f:\Libraries\AixLib\AixLib\Building\Components\Weather\RadiationOnTiltedSurface\RadOnTiltedSurf_Liu.mo
f:\Libraries\AixLib\AixLib\DataBase\Weather\WeatherBaseDataDefinition.mo
```
are UTF8 encoded without BOM. All other Mo files are plain ASCII.

**Comment by PRemmen** [comment]
thanks @tbeu for the comment. What would be your recommendation for a uniform encoding for the whole library.

**Comment by marcusfuchs** [comment]
Will be solved in #172
